gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
+#Sat Aug 10 17:06:24 IST 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
-networkTimeout=10000
-validateDistributionUrl=true
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
I don't think we need the changes to this file

Reply:
This is happening whenever I try to build the project. I will keep this in mind now.